steal.lic

```diff
@@ -161,7 +161,7 @@ class Steal
     end
 
     {
-      'new' => true, 'room' => Room.current.id, 'province' => 'zoluren',
+      'new' => true, 'room' => Room.current.id, 'steal_town' => @steal_town,#steal_town replaced province
```

Comment:
I'd like to be able to upload stealing info without changing my hometown, it feels like this should be an argument.